src/lando/utils/tasks.py

```diff
@@ -9,17 +9,23 @@
 from django.conf import settings
 from django.core import mail
 
-from lando.api.legacy.celery import celery
 from lando.api.legacy.email import make_failure_email
 from lando.api.legacy.phabricator import (
     PhabricatorClient,
     PhabricatorCommunicationException,
 )
+from lando.utils.celery import app as celery_app
 
 logger = logging.getLogger(__name__)
 
 
-@celery.task(
+@celery_app.task
+def debug_task():
+    print("hello")
+    return 1
+
+
```

Comment:
Leftover from dev?

Reply:
Yeah, it will be useful once we deploy to DEV/STAGING etc... for testing celery integration, hence why I chose to keep it in.

Reply:
@zzzeid can you add a docstring describing that, how to use it etc.